addons/account/static/src/js/tax_group.js

```diff
@@ -0,0 +1,265 @@
+            // Change tax value according to the new tax group value.
+            this._changeTaxValueByTaxGroup(this.record.data.amount_by_group.filter(elem => elem[6] == taxGroupId)[0])
+                .then(function(){
+                    this._render();
+                });
+        },
+
+        //--------------------------------------------------------------------------
+        // Handler
+        //--------------------------------------------------------------------------
+
+        /**
+         * This method is called when the user is in edit mode and 
+         * leaves the <input> field. Then, we execute the code that 
+         * modifies the information.
+         * 
+         * @param {event} ev 
+         */
+        _onBlur: function (ev) {
+            ev.preventDefault();
+            if (this.mode !== 'edit' || !this._isPurchaseDocument()) { // Trigger only in edit mode.
+                return;
+            }
+            var newValue = $(ev.target).val();
+            var taxGroupId = $(ev.target).parents('.oe_tax_group_editable').attr('data-tax-group-id');
+            this._setTaxGroups(newValue, taxGroupId);
+        },
+
+        /**
+         * This method is called when the user clicks on a specific <td>.
+         * It will check that we are in edit mode.
+         * it will then hide the edit button and display the field to be edited.
+         * 
+         * @param {event} ev 
+         */
+        _onClick: function (ev) {
+            ev.preventDefault();
+            var $myTd = $(ev.target).parents('.oe_tax_group_editable');
+            if ($myTd.length == 0) {
+                return;
+            }
+            if (this.mode !== 'edit' || !this._isPurchaseDocument()) { // Can click in edit mode only.
+                return;
+            }
+            $myTd.find('.fa-pencil').hide(); // Hide pencil.
+            // Get tax group id and its value.
+            var taxGroupId = $myTd.attr('data-tax-group-id');
+            var inputValue = this.record.data.amount_by_group.filter(elem => elem[6] == taxGroupId)[0][1];
+            inputValue = fieldUtils.format.float(inputValue, {}, { 'currency_id': this.record.data.currency_id.res_id });
+            // Add an input allows the user to change the tax group amount.
+            var $mySpan = $myTd.find('.oe_tax_group_amount_value').html('<input type="text" class="o_field_float o_field_number o_input" value="' + inputValue + '"/>');
+            $mySpan.find('.o_input').focus(); // Auto select value in the input.
+        },
+
+        /**
+         * This method is called when the user is in edit mode and pressing 
+         * a key on his keyboard. If this key corresponds to ENTER or TAB, 
+         * the code that modifies the information is executed.
+         * 
+         * @param {event} ev 
+         */
+        _onKeydown: function (ev) {
+            if (this.mode !== 'edit' || !this._isPurchaseDocument()) { // Trigger only in edit mode.
```

Comment:
This should only happen if we are already in edit mode and on a purchase document so you can remove those check